frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -271,18 +273,29 @@ object Project {
       (baseOptions ++ baseSemanticdbOptions ++ pluginOption).distinct
     }
 
+    def enableDottySemanticDB(options: List[String]) = {
```

Comment:
nit: the convention we use is lowercase db for better or worse
```suggestion
    def enableDottySemanticdb(options: List[String]) = {
```